lib/receptor_controller/client/response_worker.rb

```diff
@@ -56,6 +66,28 @@ def stop
       end
     end
 
+    # Listening for the consumer call to the Fetch API
+    # Then releasing original starting thread
+    def init_notifications(init_lock, init_wait)
+      ActiveSupport::Notifications.subscribe('request.connection.kafka') do |*args|
```

Comment:
@slemrmartin Where is this event published?

Reply:
@roliveri they are published by ruby-kafka: https://github.com/zendesk/ruby-kafka/ (chapter Instrumentation of Readme)

Reply:
Thanks, that makes things more clear.